Mathlib/Algebra/Order/Floor.lean

```diff
@@ -163,20 +163,27 @@ theorem lt_floor_add_one (a : α) : a < ⌊a⌋₊ + 1 := by simpa using lt_succ
 #align nat.lt_floor_add_one Nat.lt_floor_add_one
 
 @[simp]
-theorem floor_coe (n : ℕ) : ⌊(n : α)⌋₊ = n :=
+theorem floor_natCast (n : ℕ) : ⌊(n : α)⌋₊ = n :=
   eq_of_forall_le_iff fun a => by
     rw [le_floor_iff, Nat.cast_le]
     exact n.cast_nonneg
-#align nat.floor_coe Nat.floor_coe
+#align nat.floor_coe Nat.floor_natCast
+
+@[deprecated (since := "2024-06-08")] alias floor_coe := floor_natCast
```

Comment:
Have you heard about the new `deprecated alias` code snippet btw?

Reply:
I used it here :)